### PR TITLE
Minor fixes and cleanup

### PR DESCRIPTION
**Pull request by mumbel**
* stores should only stored their size
* BTSTL/BTSTH should operate on result like other results
* removed unnecessary masking of 4-bit number

### CODE REVIEW

File: data/languages/fr60.sinc [hunk 1]
```diff
@@ -233,17 +233,17 @@ DIR8B_REL: reloc is dir8 [ reloc = dir8 * 1; ] {
...
 
 :AND rj, @ri is op8=0x84 & rj & ri {
-    *ri = *:4 ri & rj;
+    *:4 ri = *:4 ri & rj;
     resultFlags(*:4 ri);
 }
 
 :ANDH rj, @ri is op8=0x85 & rj & ri {
-    *ri = *:2 ri & rj:2;
+    *:2 ri = *:2 ri & rj:2;
```
Comment (DiscoStarslayer) [inline data/languages/fr60.sinc:241]: thank you, I made the assumption that the target pointer would auto-size to the value being written to it. This cleans up a lot of the auto-type detecting

**Review by DiscoStarslayer — APPROVED** [review]
Great fixes, thank you!

**Comment by mumbel** [comment]
@DiscoStarslayer  oops totally forgot to do this.  Not 100% sure, but with this much changing, you might need/want to increment the version number in the ldef to like 1.1

https://github.com/desrdev/ghidra-fr60/blob/dd695fd4b15721c1101b916328ad4eef40fa94ce/data/languages/fr60.ldefs#L10

File: data/languages/fr60.sinc (hunk 1 above)
Comment (mumbel) [reply data/languages/fr60.sinc:241]: I did not check the compiled pcode, but better safe than sorry I guess

**Comment by DiscoStarslayer** [comment]
@mumbel will do, thanks. Having a few environment issues getting 10.2 to build on my machine, once I have it up and running I'll bump this and cut a release!

**Pull request merged**